contracts/instruments/RibbonThetaVault.sol

```diff
@@ -279,18 +291,51 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
      * @param share is the number of vault shares to be burned
      */
     function withdraw(uint256 share) external nonReentrant {
-        uint256 withdrawAmount = _withdraw(share);
+        uint256 withdrawAmount = _withdraw(share, false);
         IERC20(asset).safeTransfer(msg.sender, withdrawAmount);
     }
 
+    /**
+     * @notice Withdraw from V1 vault to V1 vault
+     * @notice Waive fee if registered in vault registry
+     * @param share is the number of vault shares to be burned
+     * @param vault is the address of destination V1 vault
+     */
+    function withdrawToV1Vault(uint256 share, address vault)
+        external
+        nonReentrant
+    {
+        require(vault != address(0), "!vault");
+        require(share > 0, "!share");
+
+        bool feeless = IRegistry.canWithdrawForFree(address(this), vault);
+        require(feeless, "!feeless");
+
+        uint256 withdrawAmount = _withdraw(share, feeless);
+
+        // Send assets to new vault rather than user
+        IERC20(asset).safeApprove(vault, withdrawAmount);
+        IRibbonV1Vault(vault).deposit(withdrawAmount);
+        uint256 receivedShares = IERC20(vault).balanceOf(address(this));
+        IERC20(vault).safeTransfer(msg.sender, receivedShares);
+
+        emit WithdrawToV1Vault(msg.sender, share, vault, receivedShares);
```

Comment:
We can retain the `Withdraw` event so it will show as a Withdraw and Deposit on the frontend

Reply:
Is it ok that the event won't contain information about which vault the funds are moving to?

Reply:
Ah, misinterpreted as replace `WithdrawToV1Vault` with regular `Withdraw`. Will keep both